src/backend/common/queries/dict_converters/media_converter.py

```diff
@@ -40,6 +40,7 @@ def mediaConverter_v3(cls, media: Media) -> MediaDict:
             "preferred": True if media.preferred_references != [] else False,
             "view_url": None,
             "direct_url": None,
+            "team_keys": [r.id() for r in media.references],
```

Comment:
`references` may be any key, so this should probably filter for references that are team keys.